feat: Flows, bug fixes, general improvements

khpeet opened the pull request:
- Remaining Error Budget % added - courtesy of @andrewseling 
- Renamed Indicator options within Define SLO form to be more clear + updated README accordingly
- Added `Select all` checkbox for transactions within event based indicator options: `latency (calculated)` and `availability (errors)` - #104 
- Flow/User Journey view- Ability to define more logical groupings of SLOs for event based indicators (latency/availability)- #124 
- Improved Transaction filter within Define SLO form - now uses string ‘contains’ logic and is case-insensitive - #127 
- Chart builder fixed - now passes query correctly and account id is specified.

**NOTE: Most of these changes are specific to the main component via Apps (not the entity-specific SLO/R view)**

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/nr1-slo-r?pullRequest=133) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/nr1-slo-r?pullRequest=133) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/nr1-slo-r?pullRequest=133) it.</sub>

khpeet commented:
@ricegi @andrewseling - this should be good to go now. Latest commits fix the following:

- Eslint errors
- Added "All Transactions" option and Latency (calculated) indicator to the SLO form within the entity version.
- Fixed a bug with deletion of SLOs - there wasn't any deletion of that SLO from the flow document (caused weird load behaviors)
